dessia_common/workflow/blocks.py

```diff
@@ -773,6 +773,53 @@ def _to_script(self, _) -> ToScriptElement:
         """ Write block config into a chunk of script. """
         script = f"ModelAttribute(attribute_name='{self.attribute_name}', {self.base_script()})"
         return ToScriptElement(declaration=script, imports=[self.full_classname])
+    
+    
+class GetModelAttribute(Block):
+    """
+    Fetch attribute of given object during workflow execution.
+
+    :param attribute_type: AttributeType variable that contain the model and the name of the attribute to select.
+    :param name: Name of the block.
+    :param position: Position of the block in canvas.
+    """
+
+    def __init__(self, attribute_type: AttributeType[Type], name: str = '', position: Tuple[float, float] = None):
+        self.attribute_type = attribute_type
+        inputs = [TypedVariable(self.attribute_type.class_, name=self.attribute_type.name)]
```

Comment:
`attribute_type.class_` is a str and the real class should be gotten from it with helpers' `get_python_class_from_classname` function

Reply:
```attribute_type.class_``` is a class as the screen shot shows 
![image](https://user-images.githubusercontent.com/114072906/228238702-65d1653e-3580-4721-bd67-59f65f442da0.png)

But I see your problem. I can change the ```__innit__``` function of the ```class AttributeType(Generic[T])```  from :
![image](https://user-images.githubusercontent.com/114072906/228239717-ce55aa9b-25a8-465b-a633-0f48f4cec400.png)

To:
![image](https://user-images.githubusercontent.com/114072906/228240018-c6a03a0a-09ba-4082-abae-5fa714df3353.png)

or I can do that :
![image](https://user-images.githubusercontent.com/114072906/228240424-d1ba6253-0073-458b-b02d-4db072d45812.png)

As you want. I did not see the difference and witch one would be better.